presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -15,44 +28,63 @@ The driver is also available from Maven Central:
         <version>\ |version|\ </version>
     </dependency>
 
-Requirements
-------------
+You should be using the same version of the JDBC driver as your Presto
+installation. If you are not on the latest release shown above, a list of all
+currently available versions can be found `here <https://repo1.maven.org/maven2/io/prestosql/presto-jdbc>`__.
 
-The JDBC driver is compatible with Java 8, and can be used with applications
-running on Java virtual machines version 8 and higher.
+Once downloaded, you must add the JAR file to the classpath of your Java 
+application. Often, this will be the lib folder of a Java application, but
+you should check the documentation for your specific application. Depending 
+on your application, you may be able to accomplish this from a dialog rather 
+than on the command line.  
```

Comment:
Since you do not provide any exact or template command line instructions, I would remove the sentence about dialog/cli